README.md

```diff
@@ -153,7 +151,7 @@ their Windows machine and use X11-forwarding to display the GUI.
     ```bash
     export DISPLAY=$(awk '/nameserver / {print $2; exit}' /etc/resolv.conf 2\>/dev/null):0
     export LIBGL_ALWAYS_INDIRECT=0
-    export MESA_GL_VERSION_OVERRIDE=3.3
+    export MESA_GL_VERSION_OVERRIDE=3.3 # This should no longer be needed.
```

Comment:
Is this tested?

Reply:
Only on one machine, @brenthuisman . However, I assume it will break otherwise, since we bumped OpenGL up to 4.1.

Reply:
Morning, after deliberating over this: It's actively harmful.